logprep/run_logprep.py

```diff
@@ -195,4 +197,5 @@ def main():
 
 
 if __name__ == "__main__":
+
```

Comment:
and this one I think you can remove again.

Reply:
done